src/utils/components/BootOrderModal/boot-order.scss

```diff
@@ -1,35 +1,32 @@
-.kubevirt-boot-order-summary__expandable.pf-c-expandable .pf-c-expandable__content {
-  margin-top: 0;
-}
+.kubevirt-boot-order {
+  &-order-summary__expandable.pf-c-expandable .pf-c-expandable__content {
+    margin-top: 0;
+  }
 
-.kubevirt-boot-order-summary__empty-text {
-  margin-bottom: 0;
-}
+  &-summary__empty-text {
+    margin-bottom: 0;
+  }
 
-.kubevirt-boot-order__boot-order-empty-btn {
-  margin-top: var(--pf-global--spacer--md);
-}
+  &__boot-order-empty-btn {
+    margin-top: var(--pf-global--spacer--md);
+  }
 
-.pf-c-button.pf-m-link.kubevirt-boot-order__add-device-delete-btn {
-  --pf-c-button--PaddingTop: 0;
-  --pf-c-button--PaddingRight: 0;
-  --pf-c-button--PaddingBottom: 0;
-  --pf-c-button--PaddingLeft: 0;
-  --pf-c-button--m-link--Color: var(--pf-global--Color--100);
-}
+  &__data-list-item {
+    background: var(--pf-global--BackgroundColor--100);
+  }
 
-.kubevirt-boot-order__add-device-select {
-  margin-left: var(--pf-global--spacer--lg);
-  margin-right: var(--pf-global--spacer--lg);
-}
-
-.kubevirt-boot-order__add-device {
-  display: flex;
-  padding-top: var(--pf-global--spacer--md);
-  padding-left: var(--pf-global--spacer--lg);
-  padding-right: var(--pf-global--spacer--lg);
-}
+  &__hard-disk-icon {
+    width: 18px;
+    height: 14px;
+    position: relative;
+    top: 2px;
+  }
```

Comment:
is this used somewhere? if not please remove

Reply:
right